src/OpenTelemetry.ResourceDetectors.Host/HostDetector.cs

```diff
@@ -25,9 +25,9 @@ public Resource Detect()
                 new(HostSemanticConventions.AttributeHostName, Environment.MachineName),
             });
         }
-        catch (InvalidOperationException)
+        catch (InvalidOperationException ex)
```

Comment:
nit: Add link to https://learn.microsoft.com/en-us/dotnet/api/system.environment.machinename?view=net-8.0#exceptions, so we know why we are catching this..

Reply:
7bb45fba258a11a2325331e5175912a5755931f6